app/views/foreman_rh_cloud/react/_application.html.erb

```diff
@@ -10,7 +10,6 @@
   <%= notifications %>
   <div id="organization-id" data-id="<%= Organization.current.id if Organization.current %>" ></div>
   <div id="user-id" data-id="<%= User.current.id if User.current %>" ></div>
-  <div id="ForemanInventoryUploadReactRoot"></div>
+  <div id="ForemanRhCloudReactRoot"></div>
```

Comment:
Since it's the base layout for our plugin, we can use convention here too. You can rename this file into `app/views/layouts/application.html.erb` if you want every page to have it, or `app/views/layouts/foreman_rh_cloud/react.html.erb` if you want it for this controller only.